app/models/miq_request_task.rb

```diff
@@ -139,7 +139,9 @@ def deliver_queue(req_type = request_type, zone = nil)
 
     _log.info("Queuing #{request_class::TASK_DESCRIPTION}: [#{description}]...")
 
-    if self.class::AUTOMATE_DRIVES
+    if resource_action&.configuration_script_payload
```

Comment:
configuration_script_payload is defined in a subclass.
options:
- move `configuration_script_payload` here
- move this method down
- use a `try`

least happy with `try` but it is doable

Reply:
`configuration_script_payload` is defined in `::ResourceAction` here https://github.com/ManageIQ/manageiq/blob/master/app/models/resource_action.rb#L4 so if we have a `ResourceAction` we can check `configuration_script_payload`.